### PR TITLE
Note Huawei LTE buttons

### PR DESCRIPTION
**Pull request by scop**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/91967
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/huawei_lte.markdown [hunk 1]
```diff
@@ -76,18 +77,22 @@ numbers can be set using the integration's configuration options.
 
 The following router action services are available. When invoked by a user, administrator access is required.
 
-### Service `huawei_lte.clear_traffic_statistics`
+### Deprecated service `huawei_lte.clear_traffic_statistics`
```
Comment (frenck) [inline source/_integrations/huawei_lte.markdown]: Please remove deprecated documentation instead of marking it deprecated.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/huawei_lte.markdown (hunk 1 above)
Comment (scop) [reply source/_integrations/huawei_lte.markdown]: Done. https://github.com/home-assistant/home-assistant.io/pull/27144 removes it from the fan docs whose example I followed.

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @scop :+1:

**Pull request merged**